package.json

```diff
@@ -39,10 +40,12 @@
     "portfinder": "~1.0.10",
     "request": "~2.87.0",
     "requestretry": "~2.0.2",
+    "sauce-connect-launcher": "git://github.com/saucelabs/sauce-connect-launcher.git#v1.3.1",
```

Comment:
@xwa130saucer would this require people to have ssh keys setup up in order to install the package?

Reply:
No, it doesn't. Because we host it as a public visible repo.

Reply:
According to: 
https://docs.npmjs.com/cli/install
npm install github:<githubname>/<githubrepo>[#<commit-ish>]
we are downloading repo using https protocol, thanks for pointing out!